### PR TITLE
Segota/google auth

### PR DESCRIPTION
**Pull request by CSegotaDev**
1. Fixed **GoogleAuthCallback.js**, added DEFAULT_USER data model, added comments
2. Fixed some data model reference errors, added some HTML output for data tests
3. Added **EnvironmentVariables.js** and changed API reference in **GoogleAuthCallback.js**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -63,6 +64,12 @@ function App() {
         <Route path="/main-content">
           <MainContent />
         </Route>
+        <Route path="/users/me">
+          <GoogleAuthCallback />
```
Comment (CSegotaDev) [inline src/App.js:68]: We might want to rename this Component to something that has to do with setting user data. (And I kinda just wanted to play with the review/comment functionality in PRs)

**Comment by jburger69** [comment]
Gotcha sounds good we could call it like bcUserCallback or something similar

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: src/auth/GoogleAuthCallback.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        email: currentUser.email,
+        bio: currentUser.profile.bio,
+        zipCode: currentUser.zipCode,
+        profilePictureUrl: currentUser.profilePictureUrl,
+        socialMediaLinks: currentUser.socialMediaLinks,
+        totalPoints: currentUser.totalPoints,
+        totalSeasonPoints: currentUser.totalSeasonPoints,
+        availablePoints: currentUser.availablePoints,
+        volunteerHours: currentUser.volunteerHours,
+        discordId: currentUser.discordId
```
Comment (jburger69) [inline src/auth/GoogleAuthCallback.js:59]: How are we grabbing discord Id? Does that get sent back with the google account?

**Review by jburger69 — APPROVED** [review]
Curious as to how we grab the discord id? Not sure on if that's just there or we got it through the google auth

File: src/auth/GoogleAuthCallback.js (hunk 2 above)
Comment (CSegotaDev) [reply src/auth/GoogleAuthCallback.js:59]: Yes. currentUser is the entire return from the users/me call. We then map the values that we are interested in to the DEFAULT_USER model. You can console.log(currentUser) and see the entire json object.

**Comment by CSegotaDev** [comment]
> Curious as to how we grab the discord id? Not sure on if that's just there or we got it through the google auth

The JSON object returned from users/me is going to be data from a combination of sources. Some of it is Google auth data, some of it is User (Strapi content-type), some is Profile (Strapi content-type). We'd have to look at how the users/me object is constructed in the Strapi repo to understand it exactly.

**Pull request merged**